Apps/WebClient/src/ClientApp/src/views/HomeView.vue

```diff
@@ -209,7 +209,7 @@ export default class HomeView extends Vue {
                             existingLink.filter.modules.length === 1 &&
                             existingLink.filter.modules[0] === details.type
                     ) === undefined
-            ).length === 0
+            ).length === 0 && this.showVaccineCardButton
```

Comment:
This condition won't work correctly if the "VaccinationStatus" module is disabled.

I'd suggest creating getters for the separate conditions represented by `showVaccineCardButton`: `vaccinationStatusModuleEnabled` and `preferenceVaccineCardHidden`.

Then this condition could be adjusted to `noQuickLinksToAdd && (!vaccinationStatusModuleEnabled || preferenceVaccineCardHidden)`.

Reply:
This actually was working for the "VaccinationStatus" module being disabled because it is handled in the showVaccineCardButton method. The other confusion is that this method isAddQuickLinkButtonDisabled is about disabling the button so the suggested logic above is actually backward.
However, I realize this wasn't very clear, so I changed it as you suggested in the new task (13245) to make it better creating the getters for the conditions and using them in both showVaccineStatusButton and isAddQuickLinkButttonDisabled methods.